components/TextFields/src/private/MDCIntrinsicHeightTextView.m

```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCIntrinsicHeightTextView.h"
+
+@implementation MDCIntrinsicHeightTextView
+
+/**
+ When a value in the CGSize of intrinsicContentSize is -1, it's considered undefined. For the
+ MDCMultilineTextField, we want this to always be defined so our layouts are not ambiguous.
+ */
+- (CGSize)intrinsicContentSize {
+  CGSize size = [super intrinsicContentSize];
+  if (size.height == -1) {
```

Comment:
Is -1 the same as UIViewNoIntrinsicMetric?
If so, prefer UIViewNoIntrinsicMetric.
If not, do we also need special handling of UIViewNoIntrinsicMetric?

Reply:
Correcting.